content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -605,7 +605,7 @@ webhooks:
   - operations: ["CREATE"]
     apiGroups: ["*"]
     apiVersions: ["*"]
-    resources: ["*"]
+    resources: ["*/*"]
```

Comment:
I think this is likely more correct as-is, especially since create with label selection is generally only defined for resources, not subresources. I would update the text above to say "of any resource (but not subresources)"

Reply:
Good point, I'll close out this PR then
Should we add a "with sub resources" example? This confused me a bit when I started experimenting.

Reply:
a "with status subresources" example exists in the "Matching requests: rules" section above... I'm really hesitant to give someone a copy/paste example to intercept `*/*` since that's almost always a really bad idea :)

Reply:
That's fair, my use case is pretty niche!